Userland/Services/WebDriver/Session.h

```diff
@@ -47,7 +60,10 @@ class Session {
     bool m_started { false };
     unsigned m_id { 0 };
 
-    RefPtr<WebContentConnection> m_web_content_connection;
+    unsigned next_handle_id = 0;
+    HashMap<String, NonnullOwnPtr<Window>> m_windows;
```

Comment:
Do we actually need to store windows on the heap? i.e. it seems like this could be:
`HashMap<String, Window>`